src/components/MessageBoard/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './ChatPanel';
```

Comment:
Personally I prefer to always use an `index.ts` file in folder in order to export the "public" components, hooks, methods, types, interfaces, constants, etc. from the folder. This is largely so that when importing these files, I don't need to know exactly which file the thing I'm looking for it in, but rather the overarching "type" it is (component, container, hook, store, etc.).

As well with using this format, to keep things easier, I've pretty much done away with `export default` as that would require an extra consideration in this file: `export { default as ComponentName } form './Component';`